policyengine_canada/variables/gov/provinces/ab/tax/income/benefits/acfb_base_component_post_reduction.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_canada.model_api import *
+
+
+class acfb_base_component_post_reduction(Variable):
```

Comment:
Instead of `post_reduction` in variable names and labels, use `base` and `reduction`, and then make the variable itself just something like `acfb_base_component`. This makes them more readable in the app: the total ACFB is just the ACFB base component plus the ACFB working component.